content/references/api-reference/basic-reactivity/createResource.mdx

```diff
@@ -99,6 +99,19 @@ mutate(optimisticValue);
 refetch();
 ```
 
+It's also possible to pass multiple signals as the source, by constructing a derived signal:
+
+```ts
+const [dataSignal, setDataSignal] = createSignal();
+const [moreData, setMoreData] = createSignal();
+const [data] = createResource(
+  () => [dataSignal(), moreData()],
+  ([dataVal, moreDataVal]) => {
+    // Reruns when either signal updates
```

Comment:
Thank you for adding this. Please can you add a note that when using this in typescript the values need to be explicitly typed e.g 
<img width="1034" alt="Screenshot 2023-07-15 at 15 01 47" src="https://github.com/solidjs/solid-docs-next/assets/23281128/37e92071-fd66-4d81-9f5b-f13f7b31cde2">

Reply:
Thanks for the review.
It's also possible to type this properly through `as const`: https://playground.solidjs.com/anonymous/c8210975-e41f-46ce-9488-583d1b54b686
Would it be better to use `as const` in the example instead, and state that typescript requires explicit typing?

Reply:
I've just updated it with `as const`, and why it should be included, as well as the type-assertions method as an alternative

Reply:
Great work! I'll merge this tomorrow